src/diagnostics/define_diagnostics.jl

```diff
@@ -605,6 +605,18 @@ function define_diagnostics!(land_model)
             compute_infiltration!(out, Y, p, t, land_model),
     )
 
+    # Soil albedo
+    add_diagnostic_variable!(
+        short_name = "salb",
+        long_name = "Soil Albedo",
+        standard_name = "surface albedo",
+        units = "",
+        comments = "Dependent on soil moisture",
```

Comment:
Maybe you can add a reference/equation here so that people know how it is calculated? The comment will end up in the NetCDF output

Reply:
done